src/projection.ts

```diff
@@ -0,0 +1,45 @@
+
+import {VgProjectionType} from './vega.schema';
+
+export type ProjectionType = VgProjectionType;
+
+export interface Projection {
+  /**
+   * The type of the projection. The default is "mercator".
```

Comment:
Add link to the section that describe all supported projection type?

Also use `__Default value:__` to describe default value.